docs/src/pages/customization/styled/styled.md

```diff
@@ -0,0 +1,80 @@
+# `experimentalStyled()`
+
+<p class="description">Utility for creating styled components, compatible with the Material-UI's core components.</p>
```

Comment:
What are "core components"? In the introduction section we then say "all componnents" use this utility. Is there a difference between "being compatible with" (what does that mean?) and "a component is using that utility"?

Reply:
```suggestion
<p class="description">Utility for creating styled components.</p>
```

This should be enough, later on, we explain how the util is connected to the Material-UI theme.